css/styles.css

```diff
@@ -2,27 +2,44 @@
   font-family: 'Proza Libre', sans-serif;
 }
 
+body {
+  background-color: rgb(245, 245, 245);
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
There is already a rule for the `hidden` class in `layout.css`, this can be removed.